packages/cli/index.js

```diff
@@ -1,16 +1,29 @@
 'use strict';
 
+var hopsBuild = require('hops-build');
```

Comment:
Can't we keep this in another file? I'd love to not include that express stuff if we don't actually need it.